cypress/integration/battleship_spec.js

```diff
@@ -0,0 +1,144 @@
+describe('Battleship Test', () => {
+    context('responsive', () => {
+        beforeEach(() => {
+            cy.viewport('iphone-6');
+        });
+
+        it('Opens the welcome page', () => {
+            cy.visit('/');
+            cy.contains('Welcome to the Battleship!');
+        });
+
+        it('Shows the opponent fleet', () => {
+            cy.visit('/');
+            cy.contains('Create new game').click();
+
+            cy.contains('Voir la flotte Adverse').click();
+            cy.get('.ship').should('not.exist');
+        });
+
+        it('Plays a complete game', () => {
+            let game;
+            let player1;
+            let player2;
+            let currentPlayer;
+
+            cy.request(
```

Comment:
It's not possible to use `await` instead of chained promises?

Reply:
Cypress uses [Bluebird](https://github.com/petkaantonov/bluebird) promises. WIll take a look later